app/views/katello/api/v2/content_view_components/show.json.rabl

```diff
@@ -0,0 +1,45 @@
+object @resource
+extends 'katello/api/v2/common/identifier'
+
+extends 'katello/api/v2/common/timestamps'
+
+attributes :latest
+child :composite_content_view => :composite_content_view do
+  attributes :id, :name, :label, :description, :next_version
+end
+
+child :content_view => :content_view do
+  attributes :id, :name, :label, :description, :next_version
+end
+
+child :content_view_version => :content_view_version do
+  attributes :id, :name, :label, :content_view_id, :version, :puppet_module_count
+
+  child :content_view => :content_view do
+    attributes :id, :name, :label, :description
+  end
+
+  child :environments => :environments do
+    attributes :id, :name, :label
+  end
+
+  child :archived_repos => :repositories do
+    attributes :id, :name, :label, :description
+  end
+end
+
+child :latest_version => :latest_version do
+  attributes :id, :name, :label, :content_view_id, :version, :puppet_module_count
+
+  child :content_view => :content_view do
+    attributes :id, :name, :label, :description
+  end
+
+  child :environments => :environments do
+    attributes :id, :name, :label
+  end
+
+  child :archived_repos => :repositories do
+    attributes :id, :name, :label, :description
+  end
+end
```

Comment:
Seems like these attributes can be DRYed up either by:

1) Combining content_view_version and latest_version (users should be able to look at `latest` to determine if the version is the latest version or just a content view version)
2) Or using a shared partial template (if you don't want to combine latest_version and content_view_version)